.github/workflows/pr-triage.yml

```diff
@@ -0,0 +1,41 @@
+name: Pull Request Triage
+
+on:
+  workflow_call:
+    secrets:
+      UPBOUND_BOT_GITHUB_TOKEN:
+        description: 'A PAT with permissions that allow labeling and adding PRs to a project'
+        required: true      
+
+jobs:
+  debug:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Debug
+        run: |
+          echo "github.event.pull_request.author_association: ${{ github.event.pull_request.author_association }}"
+  community-pr-triage:
+    if: ${{ (github.event.pull_request.author_association != 'OWNER' &&
+      github.event.pull_request.author_association != 'MEMBER' &&
+      github.event.pull_request.author_association != 'CONTRIBUTOR' &&
+      github.event.pull_request.author_association != 'COLLABORATOR' ) }}
+    runs-on: ubuntu-22.04
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Label as community PR
+        if: ${{ !contains(github.event.pull_request.labels.*.name, 'community') }}
+        uses: actions/github-script@v6
+        with:
+          script: |
+            github.rest.issues.addLabels({
```

Comment:
Do we label PRs using the `github.rest.issues.addLabels` API? I see the event now mentions `pull_request` (`contains(github.event.pull_request.labels.*.name...` above). I think we have already validated this.

Reply:
Yes, unfortunately, this is very confusing due to the naming but GitHub treats PRs/Issues as a similar object it seems. There isn't a corresponding `github.rest.pulls.addLabels` API.